src/mainboard/asrock/a300m-stx/src/main.rs

```diff
@@ -14,7 +14,7 @@ use model::Driver;
 use print;
 use raw_cpuid::CpuId;
 use soc::soc_init;
-// use uart::amdmmio::AMDMMIO;
+// use uart::amdmmio::UART;
```

Comment:
should we just delete this line?

Reply:
Probably. I thought about doing it, but I think Daniel is going to have to adapt A300m-stx to the new common AMD libs anyway...